src/App.vue

```diff
@@ -22,12 +22,34 @@ export default {
 }
 </script>
 
-<style>
+
+<style lang="scss">
+@import './assets/styles/custom.scss';
+@import '../node_modules/bootstrap/scss/bootstrap.scss';
 @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@100;200;400;500&display=swap');
+:root {
+  --background: #ffffff;
+  --accent: #c90e00;
+  --subtle: #6c757d;
+
+  --footer-background: #454545;
+  --footer-text: #eeeeee;
+  --footer-subtle: #888888;
```

Comment:
custom colors kann m.E. in eigene `colors.css`. Die dann wiederum in main.js eingebunden werden